src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -773,31 +768,45 @@ func (c controller) waitForOLMOperators() bool {
 		return true
 	}
 	for _, operator := range operators {
-		csvName, err := c.kc.GetCSVFromSubscription(operator.Namespace, operator.SubscriptionName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get subscription of operator %s", operator.Name)
-			continue
-		}
 
-		csv, err := c.kc.GetCSV(operator.Namespace, csvName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get %s", operator.Name)
-			continue
-		}
+		//FIXME: We check the SC of the OCS operator WA bug 1968606
+		//Remove this code when bug 1968606 is fixed
+		if operator.Name == "ocs" {
```

Comment:
We should not introduce operator specific logic in the installer even though it is temporal fix pending fix in OCS. I think we need to make the decision on the service side and have generic logic here. @rewantsoni please fix it.

Reply:
Correct we don't need any operator specific code. Could be 
sc, err := c.kc.GetStorageCluster()  is enough
and check if sc is not null then proceed, fo other operator sc will be always null,

Reply:
@pkliczewski @gobindadas I have updated it can you review it again?

Reply:
@ronniel1 Could you also review it?